### ISSUE TITLE
List supporting sites on the website

### ISSUE THREAD

**Issue by binyamin** [manton/JSONFeed#156]
t would be nice for the website to include a list of popular websites and/or news aggregators which support JSON feed. I know [NPR supports JSON feed](https://npr.codes/npr-now-supports-json-feed-1c8af29d0ce7), and [micro.blog does also](https://micro.blog/posts/manton).

**Comment by manton** [comment]
Good idea. Most of the popular feed readers support JSON Feed now, and having a full list would be useful.
